src_py/__init__.py

```diff
@@ -339,7 +339,7 @@ def __color_reduce(c):
             ver, *get_sdl_version() + sys.version_info[0:3]
         )
     )
-    print("Hello from the pygame community. https://www.pygame.org/contribute.html")
+    print("Hello from the Pygame community! To contribute, visit https://www.pygame.org/contribute.html. We appreciate you taking the time to do so.")
```

Comment:
Lowercase p is usually preferred when referring to "pygame" as per PEP8
https://peps.python.org/pep-0008/#package-and-module-names
![image](https://user-images.githubusercontent.com/35881688/209845752-f57791a4-0f41-4fd5-88e1-df4e4e423f96.png)